### PR TITLE
fix(cheatsheet): update context cmd description

### PR DESCRIPTION
**Pull request by petermikitsh**
`kubectl config view -o jsonpath='{.users[].name}'`

I noticed this command only returns the first user name, not all of them. Updating the content accordingly.

### CODE REVIEW

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ kubectl config view
 # get the password for the e2e user
 kubectl config view -o jsonpath='{.users[?(@.name == "e2e")].user.password}'
 
-kubectl config view -o jsonpath='{.users[].name}'    # get a list of users
```
Comment (xiangpengzhao) [inline content/en/docs/reference/kubectl/cheatsheet.md]: @DanyC97 guess you can confirm this as you added the line in #13257 :)

Comment (jaypipes) [reply content/en/docs/reference/kubectl/cheatsheet.md]: Actually, `.users[].name` should list all users, not just the first user. Are you sure that you don't just have a single user on the system?

Comment (petermikitsh) [reply content/en/docs/reference/kubectl/cheatsheet.md]: I have multiple users in `~/.kube.config`:

```yaml
apiVersion: v1
clusters:
current-context: snow-cluster
kind: Config
preferences: {}
users:
- name: foo
  user:
    token: abc123
- name: bar
  user:
    token: xyz456
```

`kubectl config view -o jsonpath='{.users[].name}'` outputs `bar`

`kubectl config view -o jsonpath='{.users[0].name}'` also outputs `bar`

`kubectl config view -o jsonpath='{.users[1].name}'` outputs `foo`

Comment (petermikitsh) [reply content/en/docs/reference/kubectl/cheatsheet.md]: Screen Shot 2019-08-20 at 9 58 51 AM

Comment (jaypipes) [reply content/en/docs/reference/kubectl/cheatsheet.md]: hmm, interesting. Perhaps this is a difference between `jq` and the jsonpath flag for `kubectl`

Comment (petermikitsh) [reply content/en/docs/reference/kubectl/cheatsheet.md]: @jaypipes I hope this illustrates that `get a list of users` may not be the best description of the command

Comment (jaypipes) [reply content/en/docs/reference/kubectl/cheatsheet.md]: Indeed it does! :)

**Comment by zacharysarah** [comment]
@petermikitsh Thanks for the update. This looks like it's headed in the right direction. Any reviewer can `/lgtm` once you resolve the merge conflict.

/approve

**Comment by petermikitsh** [comment]
@zacharysarah Is this PR still wanted, given https://github.com/kubernetes/website/commit/54af88fad2c3ee7f7e1fb483402c7ee0f691a1f0?

It appears the example JSON path has updated to accurately reflect the original description.

**Comment by petermikitsh** [comment]
@zacharysarah I've updated the PR to include both examples.

Since it's a cheatsheet, and people new to Kubernetes may not know both JSON path syntaxes (get first element vs. get all) -- having both is helpful.

**Pull request merged**